src/InverseSolve/misfitParam.jl

```diff
@@ -0,0 +1,164 @@
+export MisfitParam,getMisfitParam
+
+
+"""
+type jInv.InverseSolve.MisfitParam
+		
+Type storing information about one term in the misfit
+	
+F(m) = sum_i^n phi_i(pFor(model(m)),dobs,Wd)
+	
+Fields:
+	
+	pFor::ForwardProbType  - forward problem
+	Wd                     - inverse standard deviation
+	dobs                   - observed data
+	misfit::Function       - misfit function
+	modelfun::Function     - model function (evaluated locally)
+	gloc	               - mapping from inverse to forward mesh
+	   
+Constructors:
+
+getMisfitParam(pFor,Wd,dobs,misfit,model,gloc=identity)
+
+"""
+
+
+type MisfitParam
+	pFor::ForwardProbType
+	Wd
+	dobs
+	misfit::Function
+	modelfun::Function
+	gloc	
+end
+
+"""
+function jInv.InverseSolve.getMisfitParam
+		
+Required Input:	
+	
+	pFor::ForwardProbType  - forward problem
+	Wd                     - inverse standard deviation of data
+	dobs                   - observed data
+	misfit::Function        - misfit function
+	
+Optional Input:
+	
+	modelfun::Function     - model function (if all misfits have same model function, put modelfun in inverseParam and dummyMod here for efficiency)
+	gloc                   - mapping from inverse to forward mesh (default: identity)
+"""
+
+function getMisfitParam(pFor::ForwardProbType, Wd, dobs, misfit::Function, modelfun::Function=dummyMod, gloc=getGlobalToLocal(1.0))
+	return MisfitParam(pFor,Wd,dobs,misfit,modelfun,gloc);
+end
+	
+
+
+
+"""
```

Comment:
There should only be one docstring per function (not method). If there are multiple, I don't know which one is displayed when you type

?getMisfitParam

Lets combine them into one and if there are different methods, we should describe which one are optional or not.